README.MD

```diff
@@ -40,7 +40,36 @@ npm config set https-proxy http://<username>:<password>@<the.proxy.hostname>:<po
 npm config set no-proxy localhost,127.0.0.1,example.org
 ```
 
-Additionally proxy settings found in the environment variables `HTTP_PROXY`, `HTTPS_PROXY` and `NO_PROXY` will be used if they are not defined in the `.npmrc` file. 
+Additionally proxy settings found in the environment variables `HTTP_PROXY`, `HTTPS_PROXY` and `NO_PROXY` will be used if they are not defined in the `.npmrc` file.
+
+### Cache Downloaded Binaries
+By default downloaded chromium binaries are cached in a local directory which varies by operating system:
+
+* Linux: `XDG_CACHE_HOME` or ~/.cache/node-chromium/
+* macOS: ~/Library/Caches/node-chromium/
+* Windows: `LOCALAPPDATA`/node-chromium/Cache or ~/AppData/Local/node-chromium/Cache/
+* Anything else: ~/.node-chromium/
+
+You may manually add chromium archives here to avoid making network requests at all, the format of a cache entry is: `chromium-${revision}-${platform}-${arch}.zip`
+For example on macOS: `chromium-641577-darwin-x64.zip`
+
+You may override the cache path by setting the `CHROMIUM_CACHE` environment variable to a directory path, for example:
+
+```bash
+export CHROMIUM_CACHE=/path/to/cache/dir/
+
+# or in .npmrc like so:
+# chromium_cache=/path/to/cache/dir/
+```
+
+You may disable caching by setting `CHROMIUM_CACHE_SKIP` to `true`:
+
+```bash
+export CHROMIUM_CACHE_SKIP=true
```

Comment:
And the same here with the name of env var. I would suggest to replace `SKIP` with `DISABLE`. Because skip looks for me like we are still saving everything into cache but we are just skipping it when downloading/unzipping. And disable looks more like we are actually disabling the cache.
I like something like: `DISABLE_NODE_CHROMIUM_CACHE`.